alphagov/paas-billing
# Billing events to resources table

What
----

Add in a query to continuously update a resources table, sourcing from app_usage_events and service_usage_events
This currently requires to be run manually to allow for testing of new billing.

How to review
-----

- Code review
- Deploy to dev and manually add function update_resources('some_date') `\i paas-billing/billing-db/sprocs/update_resource.sql`
- Execute e.g. `SELECT update_resources('2018-01-01T00:00:00');` and see resources table filled.

Who can review
-----

not @schmie 

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
